app/views/users/circuitverse/_errors.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="contianer message-container ">
+    <% if obj.errors.any? %>
+        <div class="alert alert-danger alert-dismissible fade show" role="alert">
+            <h4 class="alert-heading">The following errors prevents User from being updated</h4>
```

Comment:
Make this text translatable using i18n

Reply:
Sure we can, but I didn't make it translatable because all the flash notices in the site are not translatable.

Reply:
> Sure we can, but I didn't make it translatable because all the flash notices in the site are not translatable.

Flash messages are translatable

Reply:
Yeah, they are what I meant was in the rest of the site flash messages don't have i18n translations.

Reply:
> > Sure we can, but I didn't make it translatable because all the flash notices in the site are not translatable.
> 
> Flash messages are translatable

Yeah, they are what I meant was in the rest of the site flash messages don't have i18n translations.